karyon3-log4j2/src/main/java/org/apache/logging/log4j/core/Log4jConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.logging.log4j.core;
+
+import org.apache.logging.log4j.core.config.xml.XmlConfiguration;
+
+public interface Log4jConfiguration {
+	
+	public void doConfigure(XmlConfiguration config);
```

Comment:
Configuration doesn't look like it has getRoot().  Would you prefer AbstractConfiguration or XmlConfiguration?